CHANGELOG.md

```diff
@@ -44,6 +47,18 @@
   - [changelog](https://github.com/getsentry/sentry-cli/blob/master/CHANGELOG.md#2421)
   - [diff](https://github.com/getsentry/sentry-cli/compare/2.41.1...2.42.1)
 
+
+### Major Changes
+
+- Remove autoSessionTracking option
+- Remove `enableTracing` Instead, set  tracesSampleRate: 0 to disable, or a higher value to enable it.
```

Comment:
Let's explain here the difference between `tracesSampleRate: undefined` and `0`.

Because `0` means the integrations are enabled and spans are created, but not sampled and `undefined` means performance integrations are disabled and no spans are created.

Reply:
From the browser changelog: https://github.com/getsentry/sentry-javascript/blob/defeaaf99ff6180be714ae770ce1ed0da6243676/CHANGELOG.md#L1667-L1672

Reply:
Added explanation here:  https://github.com/getsentry/sentry-react-native/pull/4568/commits/25bf1471978b9b24a5537ec411fc3077843e1a64